README.md

```diff
@@ -76,22 +76,22 @@ For the last mean of contribution, here are the steps to help you setup ``celer`
 1. Fork the repository and afterwards run the following command to clone it on your local machine
 
 ```shell
-git clone https://github.com/{YOUR_GITHUB_USERNAME}/celer.git
+$ git clone https://github.com/{YOUR_GITHUB_USERNAME}/celer.git
```

Comment:
this $ gets copied into the user input if ctrl c is used. for code block this is a negative point. So I'd rather not have it here